itou/templates/dashboard/dashboard.html

```diff
@@ -33,6 +33,16 @@
         </div>
     {% endif %}
 
+    {# Information pour les employeurs en cas d'absence d'annexe financière #}
+    {% if current_siae and current_siae.convention.is_active and not current_siae.convention.has_active_financial_annex %}
```

Comment:
J'ai un doute mais si la SIAE n'a pas de convention ça va pas faire une erreur ici ?

Et j'y pense que maintenant mais faut aussi limiter avec `can_show_employee_records`/`current_siae.can_use_employee_record` vu que toutes les SIAE sous conventions n'utilisent pas forcément les fiches salariés chez nous (EITI par exemple).

Reply:
bon je vais faire une variable dédiée pour éviter d'avoir 5 `and` je crois :sweat_smile: